### PR TITLE
Add Shin et al data Nat. Comm 2020

### PR DESCRIPTION
**Pull request by ococrook**
add dynamic mitoretargetting data from Shin et al

### CODE REVIEW

**Comment by ococrook** [comment]
checks and build locally with Error. Markfile warnings only

File: inst/scripts/Shin2020.R [hunk 1]
```diff
@@ -0,0 +1,166 @@
+library("MSnbase")
+library("pRoloc")
+
+
+Shin2019MitoControl <- readRDS("../../inst/extdata/Shin2019MitoCon.rds")
```
Comment (lgatto) [inline inst/scripts/Shin2020.R]: How come you have rds files here? Could you not start from csv files, or whatever initial files you have created these rds files in the first place.

**Comment by lgatto** [comment]
I have also deleted the `.Rhistory` files.

**Comment by ococrook** [comment]
I'll have to hunt down the .csv files, since they weren't generate in the CCP.

**Comment by lgatto** [comment]
Oh, that's very annoying. Are there any SI spreadsheets with the paper?

**Comment by ococrook** [comment]
I'll send some emails too see if I get the .csv, since that's probably preferable?

**Comment by lgatto** [comment]
It would be better rather than the binary files, as it is possible to trace back to something that was published. If not possible, than never mind.

**Comment by ococrook** [comment]
now using raw .csv files. No build problems locally

**Comment by lgatto** [comment]
Thanks for the PR - next time there will be 2 additional things to do
1. Bump the version (check first on [here](http://bioconductor.org/packages/devel/data/experiment/html/pRolocdata.html) to make sure you bump z by 1) and update the `NEWS` file.
2. Update the `README.md` file be re-knitting `README.Rmd`

**Comment by lgatto** [comment]
Also pushed to Bioc (devel). Will become available in a couple of days.

**Comment by ococrook** [comment]
When do these things need to be changed?

**Comment by lgatto** [comment]
Every time the package is changed, and ready to be released. 

There can of course be multiple changes/entries in the NEWS file for a single version bump.

**Pull request merged**